third_party/Elecard/MpegAudioEncLayer2/Mpegac.cpp

```diff
@@ -22,7 +22,6 @@
 #include <streams.h>
 #include <olectl.h>
 #include <initguid.h>
-#include <olectlid.h>
```

Comment:
Why are you removing this include?  I'd prefer to leave this third party code alone.

Reply:
olectlid.h just contains

#pragma message("WARNING: your code should #include olectl.h instead of olectlid.h")
#include <olectl.h>